proj/src/main/webapp/script.js

```diff
@@ -33,15 +29,20 @@ function appendMessage(payload){
 
     messagesElement.appendChild(dataHeaderElement);
     messagesElement.appendChild(dataElement);
-}
```

Comment:
reemoving a random closing bracket seems very wrong to me.

Reply:
yeah that was an accident and it broke my code so i'm fixing it right now